whitelist.py

```diff
@@ -0,0 +1,32 @@
+import re
+
+
+def write_Whitelist():
+    """
+    add words to the whitelist.txt for spelling check
+    """
+    words = dict()
+    whitelist = open(r'/Users/logan.li/project/django.txt', 'a')
```

Comment:
this path is not appropriate for other users.  A script like this should start with fetching the git repo to a cache directory, and then scanning it.
The process of building the wordlist should be included.